certbot-compatibility-test/certbot_compatibility_test/validator.py

```diff
@@ -41,7 +41,11 @@ def redirect(self, name, port=80, headers=None):
 
         redirect_location = response.headers.get("location", "")
         if not redirect_location.startswith("https://"):
-            return False
+            if not redirect_location.startswith("http://"):
```

Comment:
Maybe add a comment saying something like, "it's okay for some server configuration to redirect to an http URL, as long as it's on some other domain"?